aodntools/timeseries_products/velocity_hourly_timeseries.py

```diff
@@ -73,7 +71,7 @@ def get_resampled_values(nc_cell, ds, slice_start, varlist, binning_fun, epoch,
     ds['WCUR'][slice_start:slice_end], \
```

Comment:
mutation of input arguments in the function

Reply:
I would rename the function to highlight that I'm mutating ds.

Reply:
Renamed to `append_resampled_values`, and I've made this clear in the docstring too